propose_sns.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+#
+# This takes an optional path to a configuration file; defaults to
+# sns_init.yaml, in the same directory as this script. This argument gets passed
+# to `sns propose`. Such a file can be constructed by following the directions
+# at the top of example_sns_init.yaml.
+#
+# This is mostly copied from deploy_sns.sh.
+#
+# Besides replacing `sns deploy` with `sns propose`, the biggest differences
+# here are
+#
+#     1. No more `ic-admin ... propose-to-update-sns-deploy-whitelist`.
+#
+#     2. This does not take a CONFIG argument.
+
+set -euo pipefail
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+export SNS_CONFIGURATION_FILE_PATH="${1:-sns_init.yaml}"
```

Comment:
```suggestion
export SNS_CONFIGURATION_FILE_PATH="${1:-example_sns_init.yaml}"
```
to match the default file checked in the repository.

Reply:
The idea here is that example_sns_init.yaml should NOT be passed. Instead, the user should come up with their own sns_init.yaml, and example_sns_init.yaml can be used to "bootstrap" the sns_init.yaml creation process.

Reply:
On the other hand, to be able to run the scenario on CI (and also to give a beginner user an automated script running through the entire flow), it is beneficial if the template can be directly used (after automatically replacing aaaaa-aa with the user's DFX principal).